xtable-hudi-support/xtable-hudi-support-extensions/pom.xml

```diff
@@ -40,6 +40,12 @@
             <groupId>org.apache.xtable</groupId>
             <artifactId>xtable-core_${scala.binary.version}</artifactId>
             <version>${project.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.apache.hudi</groupId>
+                    <artifactId>hudi-common</artifactId>
```

Comment:
Another option is to declare this dependency as `provided` in this pom if we are relying on some code from `hudi-common` in this module

Reply:
Makes sense, this will reduce the dependency of other MIT/BSD dependencies as well.